src/components/Ads/Ads.tsx

```diff
@@ -0,0 +1,9 @@
+import { Image } from "mui-image";
+
+export const Ads = () => {
+  return (
+    <div>
+      <Image src="src\assets\img\ad.jpg" />
```

Comment:
The image should be dynamically imported and used in the src prop

import AdImage from 'src/assets/img/ad.jpg' for example or something like this.
And then <Image src={AdImage} alt="ad" />

Reply:
does not work